src/components/Cards/MentorCards.jsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { FaGithub } from "react-icons/fa";
+import {
+  Badge,
+  Card,
+  CardBody,
+  CardSubtitle,
+  CardText,
+  CardTitle,
+  Col,
+} from "reactstrap";
+import { GatsbyImage, getImage } from "gatsby-plugin-image";
+
+function MentorCards({
```

Comment:
This is a single card, let's name it `MentorCard`. Also requires file rename and update to all usages.


```suggestion
function MentorCard({
```

Reply:
let's do this separately as it should also be done for `Cards`